app/hotels/src/singleHotel/header/Header.js

```diff
@@ -54,7 +54,7 @@ export class Header extends React.Component<Props> {
     const photosCount = idx(hotel, _ => _.photos.edges.length) || 0;
 
     return (
-      <Touchable onPress={this.openGallery}>
+      <Touchable onPress={this.openGallery} delayPressIn={40}>
```

Comment:
Why do you add this delay?

Reply:
When we press a button it changes opacity to indicate that we are pressing it. When you put your finger on it and start scrolling, it should not show this opacity, so this delay should be sufficient to prevent that behavior.

Reply:
NB, I added a couple of more of them